junit-trait-runner/pom.xml

```diff
@@ -29,6 +29,10 @@
         <maven>3.0.5</maven>
     </prerequisites>
 
+    <properties>
+        <maven.deploy.skip>true</maven.deploy.skip>
+    </properties>
+
     <dependencies>
```

Comment:
I think we should continue to deploy the junit-trait-runner jar. It can be useful in the same way as the testutils jar.